### ISSUE TITLE
Issue with Globals Encryption 

### ISSUE THREAD

**Issue by KoMaR1911** [bluesadi/Pluto-Obfuscator#14]
Hi i'm having issue with Globals Encryption - compiler don't show any errors but strings are wrong I'm compiling python 2.7.18 lib on windows I'm using obfuscation only on 1 specific .c file
I tryed with -onlystr too but doesn't help
![qf5saX6mka](https://user-images.githubusercontent.com/15141257/161910713-503df6a8-f5f9-4467-bbeb-7688a9b50993.png)

Looking how to function looks like in IDA Dissasembler

![obraz](https://user-images.githubusercontent.com/15141257/161911097-3c62e64a-da41-4008-9720-52d2c083573e.png)

Then after all lib compiled in .exe returns random strings 

If more informations needed ask me i can provide :)

**Comment by bluesadi** [comment]
Well, that's what the GlobalsEncryption is excepted to do. This pass encrypt strings in programs at compile time and decrypt strings at run time, hiding the plain strings.

or did it break down at run time?

**Comment by KoMaR1911** [comment]
> Well, that's what the GlobalsEncryption is excepted to do. This pass encrypt strings in the program at compile time and decrypt strings at run time, hiding the plain strings.
> 
> or did it break down at run time?

Yes but problem is this is compiled to .lib then im linking it to .exe on windows i can provide .exe on Discord
